trillian/examples/deployment/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.1'
+services:
+  ctfe:
+    build:
+      context: ../../..
+      dockerfile: trillian/examples/deployment/docker/ctfe/Dockerfile
+      args:
+      - GOFLAGS
+    command: [
+      "--http_endpoint=0.0.0.0:6962",
+      "--metrics_endpoint=0.0.0.0:6963",
+      "--log_rpc_server=trillian-log-server:8090",
+      "--log_config=/run/secrets/ctfe_config",
+      "--alsologtostderr",
+    ]
+    secrets:
+    - ctfe_config
+    - ctfe_roots
```

Comment:
Why do roots count as `secrets`?

Reply:
It's the only supported way that you can provide a file via docker-compose. There's also [configs](https://docs.docker.com/v17.09/engine/swarm/configs/), but that's only supported by if you're using a [Docker swarm](https://docs.docker.com/v17.09/engine/swarm/), which we aren't.

Reply:
Ah, okey dokey.